### DISCUSSION TITLE
Get Item fields from Global Relational Variable in Insights

### DISCUSSION THREAD

**Discussion by DigNZ** [directus/directus#16362] [Archive]
Hi all,

What is the best way to get a field from the selected item when using the Global Relational Variable panel in insights? The use case is that the collection are different configurations that change values in the other panels (which days to show or thresholds etc.,)

Currently we only get ID but would like to get fields individually to get use in filters.

**Comment by br41nslug** [comment]
This is currently not possible without customization i think 🤔

**Comment by edubacco** [comment]
Hey @DigNZ , have you found a solution?

**Reply by DigNZ** [reply]
I haven't found a solution to this yet. We've started to use a custom module extension for some insights (heavily geography / map based) but I'll come back to this. We also use a custom build of Directus now in order to make changes to the in-built panels when they aren't supported. If I find a more elegant solution I'll update this issue.
